test/integration/image-component/default/pages/invalid-placeholder-blur-static.js

```diff
@@ -9,6 +9,8 @@ const Page = () => {
         id="invalid-placeholder-blur-static"
         src={testBMP}
         placeholder="blur"
+        width={200}
+        height={200}
```

Comment:
@jthcast This test should not be modified because you don't need width/height on a static import

Reply:
You are right. fixed!